client/models.go

```diff
@@ -96,18 +102,22 @@ type FeatureTemplate struct {
 	Select      SQLExpression        `json:"select"`
 	Filter      *SQLExpression       `json:"filter"`
 	Aggregate   *AggregateExpression `json:"aggregate,omitempty"`
-	PostExpr    SQLExpression        `json:"postAggregateExpr"`
+	PostExpr    *SQLExpression       `json:"postAggregateExpr"`
```

Comment:
this should probably be called postAgg rather than postExpr for consistency.

Reply:
I've gone with `PostAggExpr`. It's quite internal.